spec/services/stats/r005_monthly_performance_report_spec.rb

```diff
@@ -65,7 +65,7 @@ module Stats
           [[1, :red], [7, :red], [13, :red]],   # March
           [[1, :red], [7, :green], [13, :red]], # April
           [],
-          [[1, :red], [7, :red], [13, :red]],   # Totals
+          [[1, :red], [7, :amber], [13, :red]],   # Totals
```

Comment:
Surprised this changed from `:red` to `:amber` - is that intentional?

Reply:
Yes I think the tests previously assumed there was leftover data in the dB which contributed to the rag rating